legacy_publisher/dita_helper.py

```diff
@@ -169,3 +264,130 @@ def create_image(root=None,url=None,style=None):
     image.setAttribute('width', width)
     return image
 
+def create_dita_block(root=None,section=None, bullets=None):
+    for element in bullets:
```

Comment:
Also note:  I have a minor fear that this only handles the top level elements.  I _suspect_ we may end up needing a recursive method that calls itself for high level objects (like nested paragraphs).  We´ll see, but I´d like you to be aware of that, please.

Reply:
agree to defer this until we encounter complex content